features/practice_data.feature

```diff
@@ -10,6 +10,23 @@ Feature: The spreadsheet administrator enters practice data for a mediator under
   The practice data is entered on a single line, with each part delimited by a |. Multiple practices are delimited by
   a new line \n.
 
+  Scenario Outline: Practice data warnings
+    Given I upload a mediator with practice data <PracticeCell>
+    Then I see '<Message>'
+
+    Examples: Unmatchable tel
+      | PracticeCell                                                                   | Message                                            |
+      | 5bobbins \|15 Smith Street, London WC1R 4RL \| bish@bash.com \| http://foo.com | Record 1: Practice 1: Could not identify: 5bobbins |
+
+    Examples: Unmatchable email
+      | PracticeCell                                                                        | Message                                                  |
+      | 01234-567890 \|15 Smith Street, London WC1R 4RL \| bish(at)bash.com \| http://foo.com | Record 1: Practice 1: Could not identify: bish(at)bash.com |
+
+    Examples: Unmatchable address
```

Comment:
I'd say only one example is needed. The generic function is that we warn when a practice cell contains a string that can't be identified.